src/tpm2.c

```diff
@@ -2250,7 +2164,9 @@ TPM_RC TPM2_Quote(Quote_In* in, Quote_Out* out)
 
         TPM2_Packet_AppendAuth(&packet, ctx->authCmd, 1);
 
+        printf("\nTPM2_Quote: Packet->Pos=%d\n", packet.pos);
```

Comment:
Cleanup to remove debugging. Or if it should stay please add DEBUG_WOLFTPM around.

Reply:
Intentionally left for now, but you are right I can leave it with DEBUG_WOLFTPM even upstream. Thanks.